### DISCUSSION TITLE
Dataset Creation, Import and Create a New Experiment

### DISCUSSION THREAD

**Discussion by MananpreetKaur1** [h2oai/h2o-llmstudio#655] [Q&A]
I am new to h2o llm studio and recently was able to set it up over a virtual machine from aws ec2 instance with GPUs and NVIDIA Drivers and PyTorch pre-installed. I was just starting to setup my dataset where I read on H2o.ai about H2O LLM Data Studio and I wanted to go ahead prepare my dataset using it, but I have not found a single way to how to access it and further was not able to setup a new H2O AI Managed Cloud personal account on my own. Even on requesting for a demo as an option available on h2o.ai main website I have got no response from there. So if anyone else has used H2O LLM Data Studio can please help me on how to install/ access it can help me. Also, for the dataset I was trying to make my own dataset as I mentioned above but I have not been able to find how to name the columns or format my data in the csv so that while importing the data set the prompt and answer are read by the LLM Studio correctly.

One other issue I faced was in the create a new experiment tab on LLM Studio where I was trying with their default setup dataset to get my hands dirty with the configurations so available on the application but I wasn't able to add GPUs in the environment setup group. Although I have set the LLM Studio up on a GPU EC2 instance using AWS somehow its still not showing me the  option of any GPUs and by default it appears to be blank. Can anyone explain why is it so??????? And how can I resolve this?

In short - 1. How to access and prepare/curate documents on H2O LLM Data Studio from scratch when I don't have an H2O AI Managed Cloud account setup?
2. What is the data format for the csv file that is accepted and read correctly by the LLM Studio? Can you provide a basic sample csv with prompts and answers.
3. Why is GPUs block in create a new experiment tab coming out to be empty and not allowing me to fill it also? And if its to be filled what value am I supposed to fill it with? Or is it supposed to be filled on by default, if so how to solve this because it's not prefilled?

**Comment by MananpreetKaur1** [comment]
Can anyone help on this please?
